packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -23,6 +23,12 @@ type ChatMessageComponentAsMessageBubbleProps = {
   onEditClick: () => void;
   onRemoveClick?: () => void;
   strings: MessageThreadStrings;
+  userId: string;
+  /**
+   * Optional callback to render uploaded files in the message component.
+   * @beta
```

Comment:
Non need for beta tag as this is internally used.

Reply:
done